From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kanaka Raju Nayana <kanaka.raju.nayana@intel.com>
Date: Mon, 10 Aug 2026 21:13:31 +0530
Subject: mei: remove uuid_le_cmp

uuid_le_cmp is moved from uuid.h to mei only user.

Signed-off-by: Kanaka Raju Nayana <kanaka.raju.nayana@intel.com>
---
 drivers/misc/mei/mei_dev.h | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/misc/mei/mei_dev.h b/drivers/misc/mei/mei_dev.h
index d8634a7..e4b4793 100644
--- a/drivers/misc/mei/mei_dev.h
+++ b/drivers/misc/mei/mei_dev.h
@@ -13,11 +13,6 @@
 #include <linux/mei.h>
 #include <linux/mei_cl_bus.h>
 
-static inline int uuid_le_cmp(const uuid_le u1, const uuid_le u2)
-{
-	return memcmp(&u1, &u2, sizeof(uuid_le));
-}
-
 #include "hw.h"
 #include "hbm.h"
 
-- 
2.43.0
